cartridge/failover.lua

```diff
@@ -78,6 +78,35 @@ function _G.__cartridge_failover_wait_rw(timeout)
     return errors.pcall('WaitRwError', box.ctl.wait_rw, timeout)
 end
 
+--- Cancel all pending tasks.
```

Comment:
Is there some description what "task" is?

Reply:
I've added a comment and reorganized it once more